npm/private/utils.bzl

```diff
@@ -83,6 +84,24 @@ def _virtual_store_name(name, version):
     escaped_version = version.replace("/", "+")
     return "%s@%s" % (escaped_name, escaped_version)
 
+def _make_symlink(ctx, symlink_path, target_file):
+    files = []
+    if ctx.attr.allow_unresolved_symlinks:
```

Comment:
I'm confused about how the attr here is about unresolved symlinks, but the logic applies to resolved symlinks as well. Should something be renamed?

Reply:
It comes from `experimental_allow_unresolved_symlinks`. I think the name comes from the fact that you can have a symlink to something that is not an input. The thing you're symlinking to is not carried transitively into the sandbox or it might just not exist in or out of the sandbox at all if you pass in a path to a non-existant file.